web/modules/custom/farm_nfa/modules/farm_nfa_cfr/farm_nfa_cfr.routing.yml

```diff
@@ -0,0 +1,32 @@
+entity.cfr.management:
+  path: '/asset/{asset}/management'
```

Comment:
Why is this called management here but then the controller is named "reservetabs"?

Reply:
Or is this because management is a "shadow" default task?

Reply:
I copied how it's done for the Plan, where we have a PlanTabsController that controls the management, inventory and harvest tabs. Right now we only have the management tab on the CFR but we could have other tabs in future.
Having said that I should rename it to CfrTabsController because we always talk about CFRs, not Reserves